internal/stores/hostdb.go

```diff
@@ -334,12 +335,12 @@ func NewSQLHostDB(conn gorm.Dialector, migrate bool) (*SQLHostDB, modules.Consen
 // Host returns information about a host.
 func (db *SQLHostDB) Host(hostKey consensus.PublicKey) (hostdb.Host, error) {
 	var h dbHost
-	tx := db.db.Where(&dbHost{PublicKey: hostKey[:]}).
+	tx := db.db.Where(&dbHost{PublicKey: hostKey}).
 		Preload("Interactions").
 		Preload("Announcements").
 		Take(&h)
 	if errors.Is(tx.Error, gorm.ErrRecordNotFound) {
-		return hostdb.Host{}, ErrHostNotFound
+		return hostdb.Host{}, fmt.Errorf("host: %v; %w", hostKey, ErrHostNotFound)
```

Comment:
in this case I think it's warranted because he avoids having to check for `gorm.ErrRecordNotFound`, thus abstracting away `gorm` higher up in the stack

Reply:
returning `ErrHostNotFound` is fine, I just don't think it needs to be wrapped in `host: %v`, since the caller of this function already knows which host they're asking about.

Reply:
ah dammit I came here this morning to correct my comment 😛 but you already got to it